library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -793,6 +793,122 @@ void callSuperNotifyItemMoved(int fromPosition, int toPosition) {
         super.notifyItemMoved(fromPosition, toPosition);
     }
 
+    /**
+     * Helper method that calls {@link #notifyItemInserted} with the position of the section's
+     * header in the adapter. Useful to be called after changing the visibility of the section's
+     * header to visible with {@link Section#setHasHeader}.
+     *
+     * @param tag unique identifier of the section
+     */
+    public void notifyHeaderInsertedInSection(String tag) {
+        int headerPosition = getHeaderPositionInAdapter(tag);
```

Comment:
shouldn't we follow the pattern of calling `getValidSectionOrThrowException` then  the corresponded `notifyHeaderInsertedInSection` method with Section as parameter?